back/.rubocop.yml

```diff
@@ -43,8 +45,21 @@ Style/Documentation:
   Enabled: true
   Exclude:
   - 'db/migrate/**/*'
+Style/EmptyElse:
+  Enabled: false
```

Comment:
Why do we need empty else?

Reply:
It's hard to remember the cases why I disabled some Rubocop checks. In general, I don't like all the blue lines in my code editor and I think today the checks are really exaggerated. If I remember correctly, we discussed disabling many checks during the CDM.

In this case, I think it's because I want a function that explicitly returns false in the else clause. Rubocop suggests here to remove the else clause so nil is returned instead. In this case I disagree this would improve the code.